model/stacktrace_frame.go

```diff
@@ -13,7 +13,7 @@ type StacktraceFrame struct {
 	ContextLine *string `mapstructure:"context_line"`
 	Module      *string
 	Function    *string
-	InApp       *bool `mapstructure:"in_app"`
+	InLibrary   *bool `mapstructure:"library_frame"`
```

Comment:
I'd prefer to also call it `LibraryFrame` here as it is more consistent.

Reply:
I was trying to avoid repeating `Frame`, as in `StackFrame.LibraryFrame` but looking at it again I don't think it's a big deal and will change it.